lib/bots/versionbot.ts

```diff
@@ -1278,6 +1504,18 @@ export class VersionBot extends ProcBot {
     }
 
     /**
+     * Strip the PR author from a list of user login string.
+     *
+     * @param list          The array of user logins.
+     * @param pullRequest   The pull request to use as a base.
+     * @returns             An array containing stripped user logins, or null should there be no valid users.
+     */
+    private stripPRAuthor(list: string[] | null, pullRequest: GithubApiTypes.PullRequest): string[] | null {
+        const filteredList = (list) ? _.filter(list, (reviewer) => reviewer !== pullRequest.user.login) : null;
```

Comment:
Because I have now reached the age where certain code patterns (and style guides) are so hard-welded into my mind I don't even think about them. Shall remove. :)